### PR TITLE
chore: remove unused github client

### PR DESCRIPTION
**Pull request by sileht**

### CODE REVIEW

File: mergify_engine/github_events.py [hunk 1]
```diff
@@ -338,13 +337,13 @@ async def filter_and_dispatch(
 
 
 async def _get_github_pulls_from_sha(
-    client: github.AsyncGithubInstallationClient,
-    repo_name: str,
+    owner_login: github_types.GitHubLogin,
```
Comment (jd) [inline mergify_engine/github_events.py:340]: Would be simpler to pass a GitHubRepository entirely.

**Review by jd — CHANGES_REQUESTED** [review]

File: mergify_engine/github_events.py (hunk 1 above)
Comment (sileht) [reply mergify_engine/github_events.py:340]: We don't have one at this point.

Comment (jd) [reply mergify_engine/github_events.py:340]: Fair enough.

**Review by jd — APPROVED** [review]

**Comment by sileht** [comment]
@Mergifyio update

**Pull request merged**